azurerm/internal/services/mssql/parse/mssql.go

```diff
@@ -29,6 +29,18 @@ type MsSqlDatabaseExtendedAuditingPolicyId struct {
 	ResourceGroup string
 }
 
+func NewMsSqlDatabaseID(resourceGroup, msSqlServer, name string) MsSqlDatabaseId {
+	return MsSqlDatabaseId{
+		ResourceGroup: resourceGroup,
+		MsSqlServer:   msSqlServer,
+		Name:          name,
+	}
+}
+
+func (id MsSqlDatabaseId) ID(subscriptionId string) string {
```

Comment:
Same here, shouldn't the subscriptionId input be validated and return an error if it is null, empty, or malformed?